framework/src/outputs/DOFMapOutput.C

```diff
@@ -52,17 +52,6 @@ DOFMapOutput::DOFMapOutput(const InputParameters & parameters) :
     _system_name(getParam<std::string>("system_name")),
     _mesh(_problem_ptr->mesh())
 {
-  // Set output coloring
-  if (Moose::_color_console)
-  {
-    char * term_env = getenv("TERM");
-    if (term_env)
-    {
-      std::string term(term_env);
-      if (term != "xterm-256color" && term != "xterm")
```

Comment:
Console.C already has this.  This check was in here before the console existed/was smart enough to do it for everyone.  This is code that @dschwen suggested I remove because it was outdated and not needed.